.stylelintrc

```diff
@@ -84,11 +83,10 @@
     "selector-combinator-space-before": "always",
     "selector-list-comma-newline-after": "always",
     "selector-list-comma-newline-before": "never-multi-line",
-    "selector-list-comma-space-after": "never",
+    "selector-list-comma-space-after": "never-single-line",
     "selector-list-comma-space-before": "never",
     "selector-max-specificity": "0,4,0",
     "selector-no-id": true,
-    "selector-no-type": true,
```

Comment:
Not going to happen. This rule, even in the cases where we "must" break it, is a best practice in any good front-end development.

Disable a specific block, dismissed with an argumentative comment, to remove the unwanted warnings.